File: 0058-length-of-last-word/0058-length-of-last-word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        while(s.size() && s.back()==' ')s.pop_back();
        int cnt = 0;
        while(s.size() && s.back()!=' ')cnt++,s.pop_back();
        return cnt;
    }
};

File: 0058-length-of-last-word/0058-length-of-last-word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        s = list(s)
        while(len(s) and s[-1] == ' ') :
            s.pop()
        cnt = 0
        while(len(s) and s[-1]!= ' ') :
            s.pop();
            cnt+=1
        return cnt;